examples/VHS.hlsl

```diff
@@ -0,0 +1,95 @@
+//based on https://www.shadertoy.com/view/Ms3XWH converted by Exeldro  v 1.0
+//original by Charles 'Surn' Fettinger for obs-shaderfilter 9/2020
+//converted and made linux compatible by John Jerome 'Wishdream' Romero for obs-shaderfilter-plus 10/2020
+
+#pragma shaderfilter set range__description Range
+#pragma shaderfilter set noiseQuality__description Noise Quality
+#pragma shaderfilter set noiseIntensity__description Noise Intensity
+#pragma shaderfilter set offsetIntensity__description Offset Intensity
+#pragma shaderfilter set colorOffsetIntensity__description Color Offset Intensity
+#pragma shaderfilter set lumaMin__description Luma Minimum
+#pragma shaderfilter set lumaMinSmooth__description Luma Minimum Smooth
+#pragma shaderfilter set Alpha_Percentage__description Alpha (%)
+#pragma shaderfilter set Alpha_Percentage__min 0
+#pragma shaderfilter set Alpha_Percentage__max 100
+#pragma shaderfilter set Alpha_Percentage__slider true
+#pragma shaderfilter set Apply_To_Image__description Apply to Image
+#pragma shaderfilter set Replace_Image_Color__description Replace Image Color
+#pragma shaderfilter set Color_To_Replace__description Color to Replace
+#pragma shaderfilter set Apply_To_Specific_Color__description Apply to Specific Color
+
+uniform float range = 0.05;
+uniform float noiseQuality = 250.0;
+uniform float noiseIntensity = 0.88;
+uniform float offsetIntensity = 0.02;
+uniform float colorOffsetIntensity = 1.3;
+uniform float lumaMin = 0.01;
+uniform float lumaMinSmooth = 0.04;
+uniform float Alpha_Percentage = 100; //<Range(0.0,100.0)>
+uniform bool Apply_To_Image;
+uniform bool Replace_Image_Color;
+uniform float4 Color_To_Replace;
+uniform bool Apply_To_Specific_Color;
+
+float rand(float2 co)
+{
+    return frac(sin(dot(co.xy ,float2(12.9898,78.233))) * 43758.5453);
+}
+
+float verticalBar(float pos, float uvY, float offset)
+{
+    float edge0 = (pos - range);
+    float edge1 = (pos + range);
+
+    float x = smoothstep(edge0, pos, uvY) * offset;
+    x -= smoothstep(pos, edge1, uvY) * offset;
+    return x;
+}
+
+float4 render(float2 st)
+{
+    float2 uv = st;
+    for (float i = 0.0; i < 0.71; i += 0.1313)
+    {
+        float d = mod((builtin_elapsed_time * i), 1.7);
```

Comment:
you could use "fmod" instead of "mod"

Reply:
you know, I never thought of using fmod instead of mod, I'll add hat to the code then!